xbmc/addons/Repository.cpp

```diff
@@ -130,9 +130,9 @@ CRepository::CRepository(const AddonInfoPtr& addonInfo) : CAddon(addonInfo, Addo
   if (m_dirs.empty())
   {
     CLog::Log(LOGERROR,
-              "Repository add-on {} does not have any directory and won't be able to update/serve "
-              "addons! Please fix the addon.xml definition",
-              ID());
+              "Repository add-on {} does not have any directory matching {} and won't be able to "
+              "update/serve addons! Please fix the addon.xml definition",
+              ID(), version);
```

Comment:
probably needs to be version.asString()

Reply:
Might want to double check what the output of asString is, ive no idea what you're going to get, so may not be as useful as you were hoping. Hopefully it is though.